en/05_Contributing/05_Coding_Conventions/01_PHP_Coding_Conventions.md

```diff
@@ -6,190 +6,157 @@ iconBrand: php
 
 # PHP Coding Conventions
 
-This document provides guidelines for code formatting and documentation
-to developers contributing to SilverStripe. It applies to all PHP files
-in the `framework/` and `cms/` modules, as well as any supported additional modules.
+This document provides guidelines for code formatting and documentation to developers contributing to Silverstripe.
 
-Coding standards are an important aspect for every software project,
-and facilitate collaboration by making code more consistent and readable.
+These conventions will work well in most situations, though sometimes you may need to not follow them for practical reasons.
 
-If you are unsure about a specific standard, imitate existing SilverStripe code.
+These coding conventions are for new code, and for situations where you are intentionally refactoring code for the purpose of improving coding standards (usually as part of a major release). Existing Silverstripe CMS code is inconsistent as to whether or not it follows these conventions.
 
-## PSR-2
+## General
 
-Our goal is [PSR-2 Coding Standards](https://www.php-fig.org/psr/psr-2/) compliance.
-Since this affects existing APIs, some details like method casing will be iterated on in the next releases.
-For example, many static methods will need to be changed from lower underscore to lower camel casing. 
- 
-## Spelling
+- Aim for compliance with the [PSR-12 coding standards](https://www.php-fig.org/psr/psr-12/).
+- Prioritise the human readability of code over "clever" code that reduces the number of lines at the expense of readability.
+- All symbols and documentation should use UK-English spelling, for example "behaviour" instead of "behavior", except when necessitated by third party conventions (e.g using PHP's `Serializable` interface).
```

Comment:
```suggestion
- All symbols and documentation should use UK-English spelling, for example "behaviour" instead of "behavior" except when necessitated by third party conventions, for example using PHP's `Serializable` interface.
```

Reply:
Updated